Jenkinsfiles/web-ui-pipeline.groovy

```diff
@@ -77,16 +77,16 @@ timestamps {
             def VERSIONS_MAPPING = ['10.10': '2.4', '9.10': '2.2']
             def ELEMENTS_BASE_BRANCH = VERSIONS_MAPPING.containsKey(BASE_BRANCH) ? "maintenance-${VERSIONS_MAPPING.get(BASE_BRANCH)}.x" : BASE_BRANCH
             def MP_BASE_BRANCH = VERSIONS_MAPPING.containsKey(BASE_BRANCH) ? "${VERSIONS_MAPPING.get(BASE_BRANCH)}_${BASE_BRANCH}" : BASE_BRANCH
-            def el, uiel, webui, webuiitests, plugin
+            def elrepo, el, uiel, datavizel, webui, webuiitests, plugin
             if (params.CLEAN) {
                 sh 'npm cache clean --force'
             }
+            elrepo = cloneRebaseAndDir('nuxeo-elements', BRANCH, ELEMENTS_BASE_BRANCH)
```

Comment:
Can't we just do a single if(cloneRebaseAndDir) ... ?

Reply:
We can. I was using `elrepo` later on, but am not any more. Thanks.